scripts/add_sa_role.sh

```diff
@@ -6,7 +6,7 @@ ORG_ID=$(gcloud organizations list --format 'value(ID)')
 TIMESTAMP=$(date +%s)
 DELTA=3000
 TIMESTAMP_PLUS_EXPIRY=$((TIMESTAMP + DELTA))
-EXPIRY=$(date -ur "$TIMESTAMP_PLUS_EXPIRY" +"%Y-%m-%dT%H:%M:%SZ")
+EXPIRY=$(printf '%(%Y-%m-%dT%H:%M:%SZ)T' $TIMESTAMP_PLUS_EXPIRY)
```

Comment:
`-r` is macOS / FreeBSD specific. Since bash is already involved, this `printf` should do the trick :slightly_smiling_face: